src/jdk.jlink/share/classes/jdk/tools/jlink/resources/plugins.properties

```diff
@@ -137,6 +137,16 @@ exclude-jmod-section.usage=\
 \                            Specify a JMOD section to exclude.\n\
 \                            Where <section-name> is \"man\" or \"headers\".
 
+generate-cds-archive.argument=
+
+generate-cds-archive.description=\
+CDS plugin: generate cds archives (classes.jsa, classes_nocoops.jsa).\n\
```

Comment:
Is it necessary to show `classes.jsa, classes_nocoops.jsa` the archive file name?   Simply drop them?

Reply:
Removed the file names.